llm_studio/app_utils/sections/chat_update.py

```diff
@@ -88,6 +88,20 @@ async def update_chat_window(q):
     q.page["experiment/display/chat"].data[-1] = message
 
 
+async def chat_copy(q: Q) -> None:
+
+    chat_messages = [
+        f"{'USER' if t[1] == USER else 'BOT'}: {t[0]}"
```

Comment:
ASSISTANT is probably more common

```suggestion
        f"{'USER' if t[1] == USER else 'ASSISTANT'}: {t[0]}"
```